### ISSUE TITLE
Callback/Promise never resolve

### ISSUE THREAD

**Issue by ghost** [SamVerschueren/mongoose-seeder#9]
I'm attempting to use this module to (obviously) see my database. However, even when using the most trivial `json` file, the Promise never resolves (or rejects) and the callback is never fired if I opt-out of using promises.

Is this expected behavior?

**Comment by SamVerschueren** [comment]
Does it output something? Or nothing at all?

**Comment by ghost** [comment]
Nothing at all - my breakpoints inside `.then()` and `.catch()` are never hit. I'm not seeing any exceptions thrown either.

**Comment by ghost** [comment]
Also, to be pedantic, my database is not seeded.

**Comment by SamVerschueren** [comment]
Haven't used the lib for a while so not sure what the underlying problem can be. I'm also off to a short (5 day) vacation so unless you are able to figure it out, I'm afraid I can't be much of an assistance the coming days.

**Comment by theholla** [comment]
This just started happening to me today, as well. I started using the library with success a few weeks ago, and as of today it just lags and never resolves. I've been making changes to the seed file, so I could have caused the lag, but it's not outputting any errors and I'm not sure.

**Comment by ghost** [comment]
I've been digging into this and discovered a few potential oddities.
1. In `seed`, the `callback` is never being set to a no-op function when a Promise is being used as I would assume by the comments.
2.  There are some uncaught exceptions with Q being loaded as a module - this project is the only one where Q is being referenced - I use Bluebird elsewhere. `Q.defer()` is being created, but never resolved.

While these are not necessarily issues, they may point us in the right direction. I'll dig in deeper later today.

**Comment by ghost** [comment]
I've fixed this issue on my fork - see [Pull Request](https://github.com/SamVerschueren/mongoose-seeder/pull/10)

**Comment by SamVerschueren** [comment]
Sorry for the delay. I'm [looking for a maintainer](https://github.com/SamVerschueren/mongoose-seeder/issues/12). So if you're interested, feel free to respond.

**Comment by JeffLabonte** [comment]
@fedoranimus @SamVerschueren Was the problem fixed or the fix still has to be implemented?

**Comment by SamVerschueren** [comment]
I don't think it is. I closed the PR because I didn't have time to look into it. So feel free to reopen it and fix it.
